### ISSUE TITLE
docs.go-mono.com is broken

### ISSUE THREAD

**Issue by FavoritoHJS** [mono/website#633]
Trying to connect to http://docs.go-mono.com/ results in the browser trying to load the page for a few minutes, then timing out.

This happened since yesterday at least.

**Comment by akoeplinger** [comment]
Yes we had to turn it off, but you can try https://web.archive.org/web/20240303201523/http://docs.go-mono.com/ in case you're blocked.

I'm curious which pages you need? We should migrate those.

**Comment by FavoritoHJS** [comment]
> I'm curious which pages you need? We should migrate those.

The base class library documentation. Ideally I'd have it offline but I don't know of a way to do so...

**Comment by akoeplinger** [comment]
I'd recommend using https://learn.microsoft.com/en-us/dotnet/api/?view=netframework-4.7 instead, the BCL docs on the old mono docs site are like 15 years old.

If you go to a namespace like e.g. https://learn.microsoft.com/en-us/dotnet/api/system?view=netframework-4.7 there's also a "Download PDF" link in the table of contents.

**Comment by van800** [comment]
I have some QuickDoc/OnlineHelp functionality in Rider, which relies on docs.go-mono.com.
I wonder, is there a chance that it would be running again or I should better switch to dotnet api doc?

**Comment by akoeplinger** [comment]
We won't be bringing it back, also see https://github.com/mono/mono/issues/21796.

**Comment by Neustradamus** [comment]
Dear @mono team, @jeffschwMSFT, etc.,

It is possible to add ownership of the @mono organization to Wine HQ team? -> @julliard
It is important to have a continuous update of the code on GitHub, a mirror of https://gitlab.winehq.org/mono.

It is important to have official mirrors like the main Wine sourcecode:
- https://github.com/wine-mirror/wine (mirror of https://gitlab.winehq.org/wine/wine)

Can you give all informations of hosting servers, domain names to the new team for the continuous service?
The new team must have all to the future.

Thanks in advance.

Regards.

Initally published here:
- https://github.com/mono/mono/issues/21796#issuecomment-2335189368
